tests/ImageProcessorCore.Tests/Formats/ExifDecoderTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using ImageProcessorCore.Formats;
+using System.Linq;
+
+namespace ImageProcessor.Tests.Formats
+{
+    public class ExifDecoderTests
+    {
+
+        private TiffDecoderCore OpenJpegApp1File(string file)
```

Comment:
This is excellent stuff. I wonder whether using the static Create on the `TiffDecoder` class is confusing though. I would normally but that code in the constructor.